README.md

```diff
@@ -1,45 +1,98 @@
+|               |               IPLD Data Model               |   ipld-zcash   |          |
+|               |                                             |                |          |
++-------------------------------------------------------------+----------------+----------+
+                |                                                                         |
+                |                CID                      Path                            |
+                |                                                                         |
+                +-------------------------------------------------------------------------+
+```
 
-- ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) - this spec is a work-in-progress, it is likely not even complete.
-- ![](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square) - this spec is a rough draft and will likely change substantially.
-- ![](https://img.shields.io/badge/status-reliable-green.svg?style=flat-square) - this spec is believed to be close to final. Minor changes only.
-- ![](https://img.shields.io/badge/status-stable-brightgreen.svg?style=flat-square) - this spec is likely to improve, but not change fundamentally.
-- ![](https://img.shields.io/badge/status-permanent-blue.svg?style=flat-square) - this spec will not change.
+The goal of this stack is to enable decentralized data-structures
+which in turn will enable more decentralized applications.
 
-Nothing in this spec repository is `permanent` yet. As in many IPLD repositories, most of the work is happening in [the issues](https://github.com/ipld/specs/issues/) or in [active pull requests](https://github.com/ipld/specs/pulls/). Go take a look!
+Many of the specifications in this stack are inter-dependent.
 
-## Documents
+```
+       IPLD Dependency Graph
 
-- [**Roadmap**](/ROADMAP.md)
-- **Specifications:**
-  - ![](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square) [`IPLD`](/IPLD.md) - spec about the data model, pointers and link formats
-  - ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) `IPLD Selectors` - spec about simple language to select multiple unknown nodes in a graph
-  - ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) `IPLD Transformations` - spec about the language to trasform an IPLD graph into another
-  - ![](https://img.shields.io/badge/status-reliable-green.svg?style=flat-square) [`CID (Content IDentifier)`](https://github.com/ipld/cid)
-  - ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) [`IPLD Formats`](https://github.com/ipld/interface-ipld-format) - interface definition for adding support to different formats
-  - ![](https://img.shields.io/badge/status-draft-yellow.svg?style=flat-square) [`CAR`](/CAR.md) - Content Addressable Archives
++---+                          +-----+        +---+
+|CID+-----------+-------------->Block+-------->Raw|
```

Comment:
I still think the arrow between "Block" and "Raw" should be the other way round. A "CID" and "Raw" data form a "Block".

Reply:
Maybe I should change this to read "Raw Block." What I'm intending to reference is a full raw block, not just a CID with a "raw" codec value or just a binary blob.